pom.xml

```diff
@@ -161,7 +161,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>scm-api</artifactId>
-      <version>2.0.2</version>
+      <version>2.0.3-SNAPSHOT</version>
```

Comment:
@jglick the latest verification of plugins in core *FAILS* spectacularly with timestamp -SNAPSHOT dependencies... I have a bug to file, but right now it makes it very hard to test